CIPs/cip-0066.md

```diff
@@ -0,0 +1,113 @@
+---
+cip: 66
+title: "New Transaction Type: Celo Denominated Fees"
+author: Karl Bartel (@karlb)
+discussions-to: 
```

Comment:
Should this be set?

Reply:
Eventually yes, but we need to make a forum post first to have the link and I wanted to get this reviewed internally before we post to the forum.